### PR TITLE
Embed twitter, link to discord/reddit

### LINKED ISSUE #66: We should link our social media and discord and what not on the site.
We should link our social media and discord and what not on the site.

### PR DESCRIPTION
**Pull request by activescott**
I think fostering community participation is critical here and these are quick, high-value wins. This should close #66

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -116,6 +117,10 @@
                   <div class="tab-content">
                   <div role="tabpanel" class="tab-pane active" id="important">
                     <div style="margin: 12px;">
+                      <p>
+                        <a href="https://discord.gg/EDwd5wr" target="_blank"><img alt="Chat on Discord" src="https://img.shields.io/badge/chat-on%20discord-7289da.svg" /></a>
```
Comment (byparallax) [inline index.html]: Must be changed to **https://discord.gg/mMnRq7m**.

Comment (activescott) [reply index.html]: Doh! I will push a change for this one and the one on the readme momentarily.

Comment (activescott) [reply index.html]: Looks like the discord link in readme has already been changed in master. I am leaving that one as is. They are different from the one you requested here vs the one in the readme. I presume that is desirable. If not let me know.

**Comment by activescott** [comment]
I'll take a look at the underflow. I suspect browser-specific stuff going on here.

**Comment by activescott** [comment]
@freeatnet Should be resolved. I checked FF & Chrome. If you can take a look to confirm that would be great. I also merged in the latest theme from master and adjusted the twitter chrome for that.

**Pull request merged**